text/text.md

```diff
@@ -1620,55 +1620,55 @@ The second greatest show on earth goes to New Orleans
 So we were going back to the United States of America. What was left
 of them, 
 anyways. They still seemed all too happy to shoot the odd border
-patrol or even taking an useless pop at an airship, but was mostly
+patrol or even taking an useless pop at an airship, but were mostly
 happy to get rid of second-rate citizens, or so it seemed. 
 
 That was apparently not enough to restore the broken railway links
 between the Republic and the US. Beyond Mobile, the railroad was blown
-up to, at least, Red Bluff. So the easiest way to go from here to
+up to, at least in Red Bluff. So the easiest way to go from here to
 there, from the real Land of the Free to the purported Land of the
 Free, was to sail from Pensacola. 
 
-So it was Noah's Ark all over again, as  Boris the circus manager said at
+So it was Noah's Ark all over again, as Boris the circus manager said at
 least one zillion times before, during and after boarding. Not that we
-did not know already that all that Noah stuff was a big steaming pile
+didn't know already that all that Noah stuff was a big steaming pile
 of elephant dung, but if you put lions and horses and bearded women
 and whatnots inside a steamer they're gonna get mighty nervous, I tell
 ya. So no way Noah would have made it for forty days. Unless he got him a
-circus manager like we did. Friend of saying platitudes, but a good organizer
+circus manager like we did. Fond of saying platitudes, but a good organizer
 all the same.  
 
 Yes, I would have preferred an airship. The circus comisario was on its
 way to New Orleans already. But, pray tell, how the fuck do you expect
 to put an elephant in an airship? You did it once? Really? Why didn't
 you stick a couple of wings and a rocket up its butt and made it fly?
 No way, man, absolutely no way you could put the thirty-odd carriages
-in airships, you would have needed so many of them. Yes, you love
+in an airship, you would have needed many of them. Yes, you love
 airships, Commodore, that's a fact.
 
-This was my first chance to actually drive the mech. Aw, man, we are
-calling it a hoborg, but just from a couple of nights ago! Just let me
+This was my first chance to actually drive the mech. Aw, man, we were
+calling it a hoborg just a couple of nights ago! Just let me
```

Comment:
This was kind of the answer to someone telling him "it's a hoborg", and answering, right, but it's a recent name...